### PR TITLE
fix: redirect on logout

### PR DESCRIPTION
**Pull request by sahas-01**

### CODE REVIEW

File: src/components/AppLayout/index.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const getUserDropDown = (handleLogout: any, img?: string | null, username?: stri
 		},
 		{
 			key: 'logout',
-			label: <Link className='text-navBlue hover:text-pink_primary font-medium flex items-center gap-x-2' onClick={handleLogout} href='/'>
+			label: <Link className='text-navBlue hover:text-pink_primary font-medium flex items-center gap-x-2' onClick={(event) => handleLogout(event)} href='/'>
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:70]: remove Link from here

File: src/components/AppLayout/index.tsx [hunk 2]
```diff
@@ -353,7 +353,8 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 		setSidedrawer(false);
 	};
 
-	const handleLogout = async () => {
+	const handleLogout = async (e:any) => {
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:356]: remove e.preventDefault

**Review by KanishkaRajputd — COMMENTED** [review]
requested comments

File: src/components/AppLayout/index.tsx (hunk 1 above)
Comment (sahas-01) [reply src/components/AppLayout/index.tsx]: Resolved, pls check

File: src/components/AppLayout/index.tsx (hunk 2 above)
Comment (sahas-01) [reply src/components/AppLayout/index.tsx]: Resolved, pls check

**Review by KanishkaRajputd — APPROVED** [review]

**Pull request merged**